yarn.lock

```diff
@@ -18104,11 +18300,6 @@ typescript@^3.0.3, typescript@^3.9.7:
   resolved "https://registry.yarnpkg.com/typescript/-/typescript-3.9.7.tgz#98d600a5ebdc38f40cb277522f12dc800e9e25fa"
   integrity sha512-BLbiRkiBzAwsjut4x/dsibSTB6yWpwT5qWmC2OfuCg3GgVQCSgMs4vEctYPhsaGtd0AeuuHMkjZ2h2WG8MSzRw==
 
-typescript@^3.1.3:
```

Comment:
:-)

Reply:
I couldn't run tests otherwise. The hoisted version was incompatible to the TS code from `c/server`